ir/inst_other.go

```diff
@@ -192,7 +193,13 @@ func (inst *InstPhi) Type() types.Type {
 func (inst *InstPhi) LLString() string {
 	buf := &strings.Builder{}
 	fmt.Fprintf(buf, "%s = ", inst.Ident())
-	fmt.Fprintf(buf, "phi %s ", inst.Typ)
+	buf.WriteString("phi ")
+	for _, fmf := range inst.FastMathFlags {
```

Comment:
Minor detail, but also, for consistency, use same variable name in loop as used for fast math flags in https://github.com/llir/llvm/blob/0d19cd0df110ed9c8fbcec42e3a0239456b15e2c/ir/inst_other.go#L138

```go
	for _, flag := range inst.FastMathFlags {
		fmt.Fprintf(buf, " %s", flag)
	}
```